addons/event_sale/wizard/event_batch_configurator.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+
+
+class BatchEventConfigurator(models.TransientModel):
+    _name = 'event.event.configurator.batch'
+    _description = 'Event batch Configurator'
+
+    configurable_event_ids = fields.One2many('event.editor.line', 'editor_id')
+    sale_order_id = fields.Many2one('sale.order', 'Sales Order', required=True, ondelete='cascade')
+
+    @api.model
+    def default_get(self, fields):
+        res = super(BatchEventConfigurator, self).default_get(fields)
+        if not res.get('sale_order_id'):
+            sale_order_id = res.get('sale_order_id', self._context.get('active_id'))
+            res['sale_order_id'] = sale_order_id
+        sale_order = self.env['sale.order'].browse(res.get('sale_order_id'))
+        configurable_event_ids = []
+        for so_line in [l for l in sale_order.order_line if l.product_type == 'event']:
+            for _ in range(int(so_line.product_uom_qty)):
```

Comment:
use double underscore for unused variable (otherwise you will shadow the `_` used for translation)
```suggestion
            for __ in range(int(so_line.product_uom_qty)):
```